docs/gadget-devel/gadget-intro.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Gadget Introduction
+sidebar_position: 100
+description: Introduction to Gadgets
+---
+
+Gadgets are the central component in the Inspektor Gadget framework. A Gadget is
+an [OCI image](https://opencontainers.org/) that includes one or more eBPF
+programs, metadata YAML file and, optionally, WASM modules for post processing,
+etc. As OCI images, they use the same tooling as containers and share the same
+attributes; shareable, modular, deployable, etc.
```

Comment:
What do you mean with tooling? You mean you can use `oras`, `crane`, etc. to manipulate them? I think it's confusing because then you talk about "deployment", so I don't want users to think that they can run Gadgets with docker 😅. 

```suggestion
programs, metadata YAML file and, optionally, WASM modules for post processing.
As OCI images, they can be stored in a container registry (compliant with the
OCI specifications), making them easy to distribute and share.
```

Reply:
I took this from #3221, would you mind making the comment there? I'll align it later on.